### PR TITLE
Added Danish translation

### PR DESCRIPTION
**Pull request by MrKragh**
Not much to say, hope that helps. you. It's best effort! Some things regarding comments and technology doesn't translate well to Danish.

### CODE REVIEW

File: README.da.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+<!-- end -->
+
+---
+
+Denne README er tilgængelig på:
+
+- [Arabisk (العربية)](README.ar.md)
+- [Catalansk](README.ca.md)
+- [Dansk](README.de.md)
```
Comment (laymonage) [inline README.da.md:83]:
```suggestion
- [Dansk](README.da.md)
```

and to be added in other READMEs

File: README.da.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+[techdebtburndown]: https://techdebtburndown.com
+[contributing]: https://github.com/giscus/giscus/blob/main/CONTRIBUTING.md
+
+<!-- end -->
+
+---
+
+Denne README er tilgængelig på:
+
+- [Arabisk (العربية)](README.ar.md)
```
Comment (laymonage) [inline README.da.md:81]: We keep the languages in the list written in their respective languages so it's easy to copy-paste and sync across the READMEs

**Review by laymonage — APPROVED** [review]
Looks good, just a few minor comments but I can fix them as I merge. Tak!

**Comment by MrKragh** [comment]
My pleasure! I can see it was a bit late when I did that from your changes! Me saying thanks for this project! Let me know if I can assist with anything else.

**Pull request merged**